src/resources/vue/components/Tabs.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <div class="tabs-main-div">
+    <ul class="tabs">
+      <li
+        v-for="(tab, index) in tabs"
+        :key="index"
+        class="tab"
+        @click="changeTab(index)"
+        :class="{ active: activeTab === index }"
+      >
+        {{ tab.name }}
+      </li>
+    </ul>
+    <div class="tab-content-container">
+      <div
+        v-for="(tab, index) in tabs"
+        :key="index"
+        class="tab-content"
+        :class="{ active: activeTab === index }"
+      >
+        <item-card
+          v-for="(item, itemIndex) in tab.items"
+          :key="itemIndex"
+          :tab="tab"
```

Comment:
ipv dat je alle tabs meegeeft, Waarom geef je niet specifiek de item mee?

```js
:item="tab.items[itemIndex]"
```

Dan hoef je in de itemCard alleen maar:

`item.content`
`item.description`

etc te doen.